packages/core/src/internal/services/keycodes.service.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Maybe } from '../data-structures/maybe.js';
+import { deepClone } from '../utils/identity.js';
+
+export enum KeyCodeTypes {
```

Comment:
What do you think about using `const enum` instead of `enum`?

Reply:
Digging through it, it appears we don't need a lookup table so using `const` would be fine. In addition, I found that this enum is only used inside this file so we don't need to export it.